setup.py

```diff
@@ -1,19 +1,27 @@
 #!/usr/bin/env python3
+import os.path
 from setuptools import setup, find_packages
 
 
+# https://packaging.python.org/guides/single-sourcing-package-version/
+version = {}
+with open(os.path.join("quaternions", "version.py")) as fp:
+    exec(fp.read(), version)
+
+
 setup(
-    name='quaternions',
-    version='0.1.3',
+    name='satellogic_quaternions',
+    version=version["__version__"],
     author='Matias Graña',
     author_email='matias@satellogic.com',
-    long_description='This is a library for dealing with quaternions in python in a unified way.',
+    description='This is a library for dealing with quaternions in Python in a unified way.',
+    long_description=open('README.md').read(),
     packages=find_packages(exclude=["tests"]),
     license="?",
```

Comment:
good moment to set this as GPL v3

Reply:
Oops, this needs a rebase.

Reply:
Updated.

Reply:
I just saw this was changed in another PR, :+1: